drivers/clock_control/clock_control_mcux_sim.c

```diff
@@ -56,6 +56,7 @@ static int mcux_sim_get_subsys_rate(const struct device *dev,
 }
 
 #if DT_NODE_HAS_STATUS(DT_INST(0, nxp_kinetis_ke1xf_sim), okay)
+#define SIM_NODE DT_INST(0, nxp_kinetis_ke1xf_sim)
```

Comment:
Would be more consistent to name this `NXP_KINETIS_SIM_NODE`. I think some of the other macros could be simplified using this new macro, but that doesn't have to be done in this PR.

Reply:
Yeah, was leaving this PR to just what was needed :)